src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/misc/ModuleAutoInsult.kt

```diff
@@ -89,19 +87,20 @@ object ModuleAutoInsult : Module("AutoInsult", Category.MISC) {
         val msg = event.message
 
         // A regular expression for matching kill messages
-        val killRegex = Regex("${mc.player.name} killed (.+)")
+        val killRegex = Regex("${mc.player!!.name.literalString} killed (.+)")
```

Comment:
I would recommend to use player.nameForScoreboard, which returns the exact game profile name without any formatting and therefore is likely to be the correct name used by the server.